oauth-common/src/main/java/io/strimzi/kafka/oauth/validator/JWTSignatureValidator.java

```diff
@@ -384,7 +385,11 @@ public TokenInfo validate(String token) {
                     throw new TokenValidationException("Token validation failed: The signing key is no longer valid (kid:" + kid + ")");
                 } else {
                     // Request quick keys refresh
-                    fastScheduler.scheduleTask();
+                    try {
+                        fastScheduler.scheduleTask();
+                    } catch (RuntimeException e) {
```

Comment:
Should we catch `ServiceException` here?

Reply:
No. `ServiceException` extends `RuntimeException`, and I prefer to catch broadly here. The task scheduling error should be logged but should not have any other side effects as that would result in a different error message that is returned to the client doing the authenticating.